frontend/src/metabase/admin/upsells/UpsellSdkLink.module.css

```diff
@@ -0,0 +1,3 @@
+.link:hover {
+  text-decoration: underline;
+}
```

Comment:
I think you can just get this from `frontend/src/metabase/css/core/link.module.css`